renku/api/client.py

```diff
@@ -26,10 +26,7 @@
 
 @attr.s
 class LocalClient(
-    PathMixin,
-    StorageApiMixin,
-    RepositoryApiMixin,
-    DatasetsApiMixin,
+    PathMixin, StorageApiMixin, RepositoryApiMixin, DatasetsApiMixin
```

Comment:
are you using yapf with the style file in this repo?

Reply:
Yes.